test/xml-generator/test.py

```diff
@@ -115,5 +115,21 @@ def test_nonfatalError(self):
         tester = Tester(self.nominal_pfconfig, vector)
         self.assertTrue(tester.check(self.nominal_reference, expectedErrors=1))
 
+    def test_conflicting(self):
+        vector = TestVector(edds=[os.path.join(vector_dir, "conflicting.pfw")])
+
+        tester = Tester(self.nominal_pfconfig, vector)
+        self.assertTrue(tester.check(expectedErrors=1))
+
+    def test_fail_criteria(self):
+        pfconfig = copy.copy(self.nominal_pfconfig)
```

Comment:
- deep copy ?
 - Would it not be better to have nominal_pfconfig a NON static variable to avoid all those copy.